client/tasks/fills/shipping/rates.js

```diff
@@ -229,80 +307,34 @@ class ShippingRates extends Component {
 										className="woocommerce-shipping-rate"
 										key={ zone.id }
 									>
-										<div className="woocommerce-shipping-rate__icon">
-											{ zone.locations ? (
-												zone.locations.map(
-													( location ) => (
-														<Flag
-															size={ 24 }
-															code={
-																location.code
-															}
-															key={
-																location.code
-															}
-														/>
-													)
-												)
-											) : (
-												// Icon used for zones without locations or "Rest of the world".
-												<Icon icon={ globe } />
-											) }
-										</div>
+										<ShippingRateIcon zone={ zone } />
 										<div className="woocommerce-shipping-rate__main">
-											{ zone.toggleable ? (
-												<label
-													htmlFor={ `woocommerce-shipping-rate__toggle-${ zone.id }` }
-													className="woocommerce-shipping-rate__name"
-												>
-													{ zone.name }
-													<FormToggle
-														id={ `woocommerce-shipping-rate__toggle-${ zone.id }` }
-														{ ...getInputProps(
-															`${ zone.id }_enabled`
-														) }
-													/>
-												</label>
-											) : (
-												<div className="woocommerce-shipping-rate__name">
-													{ zone.name }
-												</div>
-											) }
-											{ ( ! zone.toggleable ||
-												values[
-													`${ zone.id }_enabled`
-												] ) && (
-												<TextControlWithAffixes
-													label={ __(
-														'Shipping cost',
-														'woocommerce-admin'
-													) }
-													required
-													{ ...getInputProps(
-														`${ zone.id }_rate`
-													) }
-													onBlur={ () => {
-														setTouched(
-															`${ zone.id }_rate`
-														);
-														setValue(
-															`${ zone.id }_rate`,
-															this.getFormattedRate(
-																values[
-																	`${ zone.id }_rate`
-																]
-															)
-														);
-													} }
-													prefix={ this.renderInputPrefix() }
-													suffix={ this.renderInputSuffix(
-														values[
-															`${ zone.id }_rate`
-														]
-													) }
-													className="muriel-input-text woocommerce-shipping-rate__control-wrapper"
```

Comment:
The root cause is the **className** from `getInputProps` is overwritten by this class name. I fixed this in line 59.